docs/how-to/command-line-quick-start.md

```diff
@@ -12,36 +12,37 @@ If you're command-line savvy and just want to get up and running with IPFS right
 Don’t want to use the command line right now? Give the desktop-app implementation of IPFS a try. It also does all the steps listed on this page automatically, so you can run IPFS from the terminal later whenever you want. [Download IPFS Desktop now](https://github.com/ipfs-shipyard/ipfs-desktop)
 :::
 
-## Install IPFS
+## Prerequisites
 
-Installing IPFS is simple, but varies between operating system:
+If you have not yet installed go-ipfs follow the instructions below to install it.
```

Comment:
Instead of listing the install instructions, can we link to the [existing instructions](https://docs.ipfs.io/install/command-line/) instead?

Reply:
Replaced the instructions with a link the existing instructions.

Reply:
Excellent.